Build success but README.md not getting updated with new images.

**Describe the bug**
Build success but README.md not getting updated with images.

**To Reproduce**
Steps to reproduce the behavior:
1. Trigger github actions on a successful pull request

**Expected behavior**
New images with description should appear on the README.md

**Screenshots**
NA

**Desktop (please complete the following information):**
 NA

**Smartphone (please complete the following information):**
NA

**Additional context**
[Checkout this successful build which should have updated the readme markdown file ](https://github.com/bongdevhq/profile-showcase/actions/runs/6481128361)

This failed with following 

`File "/home/runner/work/profile-showcase/profile-showcase/.github/scripts/update_readme.py", line 20
    image = lines[2].split(":")[1].strip().replace("'", "")                location = lines[2].strip().split(":")[1].strip().replace("'",")
                                                                                                                                         ^
SyntaxError: unterminated string literal (detected at line 20)
Error: Process completed with exit code 1.
`

@ItsRoy69 @chiraagbanerjee

--- a/.github/scripts/update_readme.py
+++ b/.github/scripts/update_readme.py
@@ -13,26 +13,26 @@ for filename in os.listdir(profiles_dir):
                 print(f"Warning: File {filename} has fewer lines than expected. Skipping...")
                 continue
             if len(lines[0]) > 9:  # line 0 should not contain any information it's only for markup
-                print(f"Warning: File {filename} doesn't match the expected format. Skipping...")
+                print(f"Warning: File {filename} doesn't match the expected format.Skipping...")
                 continue
             try:
+                
                 name = lines[1].split(":")[1].strip().replace("'", "")
-                image = lines[2].split(":")[1].strip().replace("'", "")                location = lines[2].strip().split(":")[1].strip().replace("'",")
-                bio = lines[3].strip().split(":")[1].strip().replace("'",")
-                institution = lines[4].strip().split(":")[1].strip().replace("'",")
-                organisation = lines[5].strip().split(":")[1].strip().replace("'",")
-
+                image = lines[2].split(":")[1].strip().replace("'", "")                
+                location = lines[3].strip().split(":")[1].strip().replace("'","")
+                bio = lines[4].strip().split(":")[1].strip().replace("'","")
+                institution = lines[5].strip().split(":")[1].strip().replace("'","")
+                organisation = lines[6].strip().split(":")[1].strip().replace("'","")
+                
                 # Resizing the image to a smaller version
                 im = Image.open(f"static/images/{image}")
                 im.thumbnail((250, 250))
                 im.save(f"static/images/{image}", "JPEG")
-
-                profiles.append((name, image, location, bio, institution, organisation))
+                
+                bio = lines[4].split(":")[1].strip().replace("'", "")
+                profiles.append((name, image,location, bio,institution,organisation))
             except IndexError:
                 print(f"Warning: File {filename} doesn't match the expected format. Skipping...")
-                continue
-            except Exception as e:
-                print(f"Error processing file {filename}: {str(e)}")
 
 with open(readme_path, 'r') as f:
     readme_content = f.read()
@@ -41,8 +41,8 @@ start_placeholder = "## Contributors:\n\n"
 end_placeholder = "## Happy Hacking! 🚀\n"
 profile_strings = []
 
-for name, image, location, bio, institution, organisation in profiles:
-    profile_md = f"![{name}](static/images/{image})\n\n**{name}**\n\nLocation: {location}\n\nBio: {bio}\n\nInstitution: {institution}\n\nOrganisation: {organisation}\n\n---\n\n"
+for name, image,location, bio,institution,organisation in profiles:
+    profile_md = f"![{name}](static/images/{image})\n\n**{name}**\n\n{bio}\n\n**{institution}**\n\n{organisation}\n\n{location}\n\n---\n\n"
     profile_strings.append(profile_md)
 
 new_content = readme_content.split(start_placeholder)[0] + start_placeholder + "\n".join(profile_strings) + end_placeholder

--- a/.github/workflows/update_readme.yml
+++ b/.github/workflows/update_readme.yml
@@ -28,4 +28,10 @@ jobs:
       run: python -m pip install Pillow
 
     - name: Update README
-      run: python .github/scripts/update_readme.py
+      run: |
+        python .github/scripts/update_readme.py
+        git add README.md static/images content
+        git config --local user.email "bong.dev@github.com"
+        git config --local user.name "bong.dev"
+        git commit -m "Update README.md"
+        git push

